liquibase/src/main/java/org/openmrs/liquibase/Main.java

```diff
@@ -9,23 +9,24 @@
  */
 package org.openmrs.liquibase;
 
-import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
+
```

Comment:
just some little unwanted spaces

Reply:
@HerbertYiga that's alright. java imports are grouped at the top, then javax, then org, then com. see https://wiki.openmrs.org/display/docs/Install+and+Configure+Eclipse+For+OpenMRS+Development#InstallandConfigureEclipseForOpenMRSDevelopment-ConfigureJavaImportOrder